src/frontend/Module.php

```diff
@@ -42,6 +42,7 @@
      * @var array
      */
     public $urlRules = [
+        ['pattern' => 'news/<category>/<id:\d+>-<title>', 'route' => 'news/default/detail'],
```

Comment:
Maybe we should be consistent with the naming along with the other rule:

`news/<category>/<id:\d+>/<title>` with title you mean the alias/slug?

Reply:
Yes, but I made it so module does not break if there is no article slug provided - `news` module will continue using generated title for URL